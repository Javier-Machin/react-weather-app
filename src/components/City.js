import React from 'react';
import CityVector from '../images/weather-city.jpg';

// City background

const City = () => {
  return (
    <img
      className="city-vector"
      src={CityVector}
      alt="Drawing of some buildings representing a city"
    />
  );
};

export default City;
