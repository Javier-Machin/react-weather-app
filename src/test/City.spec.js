import React from 'react';
import { shallow } from 'enzyme';
import City from '../components/City';

describe('City', () => {
  it('should render correctly', () => {
    const component = shallow(<City />);

    expect(component).toMatchSnapshot();
  });
});
