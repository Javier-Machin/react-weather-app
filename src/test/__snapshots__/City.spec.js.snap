// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City should render correctly 1`] = `
<img
  alt="Drawing of some buildings representing a city"
  className="city-vector"
  src="weather-city.jpg"
/>
`;
